Documentation/CodeSnippets/TypoScript/Part1FluidTemplateSection.rst.txt

```diff
@@ -0,0 +1,36 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/TypoScript/Setup/Part6ProcessedContent
+   :linenos:
+
+   
+   page {
+     10 {
+       dataProcessing {
+         //10 = TYPO3\CMS\Frontend\DataProcessing\MenuProcessor
```

Comment:
Why commented?

Reply:
The comments are used to show where in the larger context of the code we are working